detailed-design/pobi.md

```diff
@@ -1,86 +1,349 @@
+### Aggregator Incentives
 
-#### Incentives For Publishing
-The high level goal is to keep the system functioning as smoothly as possible, and be resistant to random failures or malicious behaviour, while not penalising Obscuro nodes for not being available.
+Compared to a typical L1 protocol, there is an additional complexity. In a L1 like Bitcoin or Ethereum, once a node
+gossips a valid block, all the other nodes are incentivised to use it as a parent, because they know everyone will do
+that as well. In a L2 decentralised protocol like POBI, there is an additional step, which is the publication of the
+rollup to L1, which can fail for multiple reasons.
 
-The reward mechanism implements the following rules:
-1. Pay an attractive reward to the winner of the round if they publish the rollup such that it is included in the next Ethereum block.
-2. Pay a minimal reward if it is included with one block delay. This reward should only cover the cost of the gas.
-3. Do not pay anything to a competing aggregator that front-runs the winner, and publishes an alternative non-winning rollup. Considering that the frontrunner is consuming precious Ethereum gas, this mechanism is in effect a punishment.
-   This case will manifest itself as two competing rollups for the same generation published in sequential Ethereum blocks, where the rollup published in the first block has a higher nonce.
+The high level goal is to keep the system functioning as smoothly as possible, and be resistant to random failures or
+malicious behaviour, while not penalising Obscuro nodes for not being available.
 
-_Note that in the latter case, to achieve smooth running, the non-winning rollup published by the frontrunner is part of the canonical chain, but it pays no reward._
+These are the aggregator rewarding rules:
 
-As a consequence of these rules, any winning aggregator is incentivised to publish with the _gas_price_ high enough to claim the full reward.
+1. The aggregator that has a single rollup without competition included in a block that was generated with the proof of
+   the parent L1 block, will get the full reward. This is the most efficient case that is encouraged.
 
-The reward rules are depicted in the following diagram:
-![L1 front running](../images/block-rewarding.png)
+2. If multiple rollups generated with the same L1 proof and different nonces are published in the same block, the one
+   with the lowest nonce is on the canonical chain, but the reward is split between them in a proportion of 80/20 (this
+   ratio is indicative only). The reason for this rule is that it incentivises aggregators to gossip their winning
+   rollup such that no one else publishes at the same time.
+    - There is no incentive for the losing aggregator to publish as 20% of the reward will not even cover the cost of
```

Comment:
Who says 20% won't cover gas? Doesn't it depend on the reward size and the gas price?

Reply:
Fair point. 
The assumption is that the rewards for the aggregators won't be that high, as we don't want to fleece the users.